.github/workflows/pano-deploy.yml

```diff
@@ -1,6 +1,8 @@
 name: Deploy Pano
 on:
   push:
+    paths:
+      - apps/pano/**
```

Comment:
we also need to add the packages it depends on. eg `kampus-ui` => meaning add `- packages/kampus-ui/**` here